README.md

```diff
@@ -19,6 +19,21 @@ functions for platform with support for malloc() and BSD-style
 sockets. The examples directory contains a client and a server to
 demonstrate the use of this library. 
 
+INSTALLATION INSTRUCTIONS - BRANCH DTLS
+================
+
+First get the TinyDTLS submodule:
+```bash
```

Comment:
There is nothing bash-specific in the following commands, so I presume this could be "shell" here?